### ISSUE TITLE
Build failure with IM 7.1.1-25

### ISSUE THREAD

**Issue by remicollet** [Imagick/imagick#645]
```
========DIFF========
001+ Fatal error: Uncaught Exception: Value for kurtosis doesn't match expected. Expected: -0.93792610350105, actual: -0.7092599567492 in /dev/shm/BUILD/php-pecl-imagick-im6-3.7.0/imagick-3.7.0/tests/functions.inc:132
002+ Stack trace:
003+ #0 /dev/shm/BUILD/php-pecl-imagick-im6-3.7.0/imagick-3.7.0/tests/316_Imagick_getImageKurtosis.php(9): check_value(Array, 'kurtosis', -0.93792610350105)
001- Value for 'kurtosis' is %f which is close enough to expected %f
002- Value for 'skewness' is %f which is close enough to expected %f
003- Ok
004+ #1 /dev/shm/BUILD/php-pecl-imagick-im6-3.7.0/imagick-3.7.0/tests/316_Imagick_getImageKurtosis.php(13): getImageKurtosis()
005+ #2 {main}
006+   thrown in /dev/shm/BUILD/php-pecl-imagick-im6-3.7.0/imagick-3.7.0/tests/functions.inc on line 132
========DONE========
FAIL Test Imagick, getImageKurtosis [tests/316_Imagick_getImageKurtosis.phpt] 

```

**Comment by Danack** [comment]
This looks like a deliberate change in the calculations: https://github.com/ImageMagick/ImageMagick/issues/6924


So I'll update the expectations of the tests.

**Comment by Danack** [comment]
Fixed in 944b67fce68bcb5835999a149f917670555b6fcb
